src/main/java/mate/academy/springintro/model/Category.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.springintro.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.Where;
+
+@Entity
+@Getter
+@Setter
+@Where(clause = "is_deleted = FALSE")
+@Table(name = "category")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    private String description;
+
+    @Column(nullable = false, name = "is_deleted")
+    private Boolean deleted = false;
```

Comment:
```suggestion
    private boolean deleted = false;
```
Don’t use Wrapper. It can provide `null` value.